### PR TITLE
Feature/brpa 757 industry select

### PR DESCRIPTION
**Pull request by ConradoClark**
#### What is the purpose of this pull request?
- add a grouped dropdown component to allow selecting the industry.

#### What problem is this solving?
- It's one of the options the client must select before integrating with TikTok

#### How to test it?
[In this workspace.](https://tiktok--sandboxbrdevbrl.myvtex.com/admin/tiktok)

#### Screenshots or example usage
tiktok_industry

#### Types of changes

- [ ] Chore (non-breaking change which doesn't change any functionalities)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/StartupForm.tsx [hunk 1]
```diff
@@ -1,16 +1,52 @@
...
+  return (
+    <Formik onSubmit={handleSubmit} initialValues={initialValues}>
+      <Form>
+        <FormikGroupedSelect
+          name="industry"
+          label="Industry"
+          options={industryOptions}
+        />
+        <Button type="submit" className="mt4">
+          Submit
```
Comment (MIzidio) [inline react/components/StartupForm.tsx:45]: This string shouldn't be in a message file ?

Comment (ConradoClark) [reply react/components/StartupForm.tsx:45]: Yes it should! But I'm going to do that on another PR, this submit button is just a dummy for now 😅

**Review by felipefadul — APPROVED** [review]
Good! I only left a suggestion to format the changelog file.

**Comment by ConradoClark** [comment]
This is weird. I think there must have a separate entry for each commit or else the `danger` bot complains about changelog standards, @felipefadul. I think I need to rollback your suggestion 🤔

**Comment by felipefadul** [comment]
That's ok! I usually ignore this bot complaint when there's a stacked PR. :sweat_smile:
Do you know if the bot will merge the various `Added` sections of the changelog into one after publishing the version?

**Review by MIzidio — APPROVED** [review]

**Pull request merged**